translations_auto_pr.js

```diff
@@ -0,0 +1,215 @@
+'use strict'
+
+const fs = require("fs-extra");
+const cp = require("child_process");
+const Octokit = require('@octokit/rest')
+const path = require('path');
+const parseGitConfig = require('parse-git-config');
+
+const branchName = 'localization';
+const mergeTo = 'main';
+const commitComment = 'Localization - Translated Strings';
+const pullRequestTitle = '[Auto] Localization - Translated Strings';
+
+let repoOwner = process.argv[2];
+let repoName = process.argv[3];
+let authUser = process.argv[4];
+let authToken = process.argv[5];
+let userFullName = process.argv[6];
+let userEmail = process.argv[7];
+let locRootPath = process.argv[8];
+let locSubPath = process.argv[9];
+
+if (!repoOwner || !repoName || !authUser || !authToken || !userFullName || !userEmail || !locRootPath || !locSubPath) {
+    console.error(`ERROR: Usage: ${path.parse(process.argv[0]).base} ${path.parse(process.argv[1]).base} repo_owner repo_name auth_token user_full_name user_email loc_root_path loc_sub_path`);
+    console.error(`   repo_owner - The owner of the repo on GitHub.  i.e. microsoft`);
+    console.error(`   repo_name - The name of the repo on GitHub.  i.e. vscode-cpptools`);
```

Comment:
Maybe update to refer to `vscode-makefile-tools`?

Reply:
Ah how I missed that! Of course, all scripts I copied from cmake or cpp tools I updated wherever it was obvious. But I missed this one, great catch, thank you.